sdcm/logcollector.py

```diff
@@ -102,10 +102,17 @@ class BaseLogEntity:  # pylint: disable=too-few-public-methods
     collect_timeout = 300
     _params = {}
 
-    def __init__(self, name, command="", search_locally=False):
+    def __init__(self, name, command="", search_locally=False, collect_to_parent=False):
+        """
+         collect_to_parent: relevant for cluster:
```

Comment:
```suggestion
         collect_from_parent: relevant for cluster:
```

or `collect_from_cluster_folder`

otherwise the name is a bit unclear

Reply:
this code is collect, and it collects from a directory.

the save code, saves to some specific folder.

I think the terminology should go after whom currently doing the actions